packages/account/src/Containers/Account/account.tsx

```diff
@@ -31,16 +31,30 @@ const Account = observer(({ history, location, routes }: TAccountProps) => {
         should_allow_authentication,
         should_allow_poinc_authentication,
     } = client;
-    const { toggleAccountSettings, is_account_settings_visible } = ui;
+    const { toggleAccountSettings, is_account_settings_visible, is_mobile, is_desktop } = ui;
+
+    const [available_routes, setAvailableRoutes] = React.useState(routes);
+    const { is_passkey_supported, is_passkey_support_checking } = useIsPasskeySupported();
+
+    const should_remove_passkeys_route = is_desktop || (is_mobile && !is_passkey_supported);
```

Comment:
should we check if `is_passkey_support_checking` before removing passkey route?
what if checking is still in progress?

Reply:
we do it, but I got your point. will move should_remove_passkeys_route variable in useEffect